src/bundle/Resources/views/content/tabs/locations.html.twig

```diff
@@ -1,7 +1,11 @@
 {% trans_default_domain "locationview" %}
+{{ form_start(actionsForm, {'action': path('ez_hybrid_platform_ui_location_actions', {'contentId': content.id})}) }}
 
 <div class="ez-list-toolbar">
     <h2 class="ez-list-toolbar-label">{{ 'locationview.locations.content.locations'|trans()|desc('Content locations') }}</h2>
+    <div class="container-buttons">
+        {{ form_row(actionsForm.delete, {'label': 'locationview.locations.delete.locations'|trans()|desc('Delete selected locations'), 'attr': {'class': 'ez-button ez-button-negative'}}) }}
```

Comment:
Should use `form_widget` for this rather than `form_row` so that it only outputs the button without the surrounding div